internal/venice-common/src/main/java/com/linkedin/venice/pubsub/api/PubSubTopicType.java

```diff
@@ -22,7 +21,7 @@ public static PubSubTopicType getPubSubTopicType(String topicName) {
     } else if (VeniceView.isViewTopic(topicName)) {
       return VIEW_TOPIC;
     } else {
-      throw new VeniceException("Unsupported topic type for: " + topicName);
+      return UNKNOWN_TYPE_TOPIC;
```

Comment:
Is there never a place in the code where we care to look for this? e.g. in the topic cleaner or elsewhere, would we want to skip unknown topics?

Reply:
Inside topic cleaner, we only care about versioned, real-time topics. But I am still wondering do we need to add some filtering logic inside TopicManager to avoid bring in non-venice type topics? as even venice-only kafka cluster could contain non-venice type topics.